nebula-logger/main/logger-engine/objects/LogEntryEvent__e/fields/TimestampInMilliSecond__c.field-meta.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>TimestampInMilliSecond__c</fullName>
```

Comment:
Minor detail, but for consistency, can you make the 'S' in 'MilliSecond__c' lowercase (i.e., `TimestampInMillisecond__c`)?

Reply:
@jongpie Appreciate your detail for consistency. Fixed the field name. Let me know.